modin/pandas/test/test_io.py

```diff
@@ -1352,7 +1352,7 @@ class TestParquet:
         reason="The reason of tests fail in `cloud` mode is unknown for now - issue #3264",
     )
     def test_read_parquet(self, make_parquet_file, columns, row_group_size, engine):
-        unique_filename = get_unique_filename(extension="parquet")
+        unique_filename = Path(get_unique_filename(extension="parquet"))
```

Comment:
Wouldn't it better to have both types (`str/Path`) of `unique_filename` (by adding type of path as test parameter)?

Reply:
Yes it would, I'll add it as a parameter.